packages/commonwealth/client/scripts/models/IWebWallet.ts

```diff
@@ -9,7 +9,7 @@ interface IWebWallet<AccountT extends { address: string } | string> {
   enabled: boolean;
   enabling: boolean;
   accounts: readonly AccountT[];
-
+  api?: any;
```

Comment:
Should be a generic arg like `ApiT`.

Reply:
I think I will need a tutorial from you about this